src/server/player_sao.cpp

```diff
@@ -146,11 +146,6 @@ std::string PlayerSAO::getClientInitializationData(u16 protocol_version)
 	return os.str();
 }
 
-void PlayerSAO::getStaticData(std::string * result) const
-{
-	FATAL_ERROR("Obsolete function");
```

Comment:
This message was misleading, this function is not deprecated or obsolete but it must never be called for PlayerSAO.
So, please keep the `FATAL_ERROR` implementation.

Reply:
I changed the message, hope it's fine